content/en/metrics/custom_metrics/type_modifiers.md

```diff
@@ -78,7 +78,7 @@ Depending on the metric type you applied them to, the behavior differs:
 
 ### The `weighted()` modifier
 
-Tags such as `pod name` or `container_name` cause high tag churn, especially when creating queries for cost management, capacity planning, or autoscaling for containerized applications. To ensure mathematical accuracy of queries on gauges regardless of tag churn, you can use a `.weighted()` in-application modifier. The `.weighted()` modifier enables Datadog to properly weight metric values based on the lifespan of these frequently churning tags. 
+Tags such as `pod name` or `container_name` cause high tag churn, especially when creating queries for cost management, capacity planning, or autoscaling for containerized applications. To ensure mathematical accuracy of queries on gauges regardless of tag churn, you can use the `.weighted()` in-application modifier. The `.weighted()` modifier tells Datadog to weight metric values based on the lifespan of these frequently churning tags. 
```

Comment:
What does "high tag churn" mean?

Also, are "pod name" and "container_name" literal tags, or are they placeholders?